components/concept/storage/build.gradle

```diff
@@ -22,6 +22,8 @@ android {
 }
 
 dependencies {
+    api Dependencies.mozilla_places
```

Comment:
`concept` components should only depend on other concept components, or ideally nothing at all. See my comment in concept-storage/BookmarksStorage